src/main/java/com/github/firmwehr/gentle/output/UserOutput.java

```diff
@@ -0,0 +1,72 @@
+package com.github.firmwehr.gentle.output;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.fusesource.jansi.Ansi.ansi;
+
+@SuppressWarnings("UseOfSystemOutOrSystemErr")
+public final class UserOutput {
```

Comment:
Why is this a singleton instead of just a few static methods?

Reply:
I second that